src/cf-layout/src/organisms/wells.less

```diff
@@ -0,0 +1,16 @@
+.o-well {
+    box-sizing: border-box;
+
+    // 8 columns plus gutters
+    max-width: unit( 770px / @base-font-size-px, em );
```

Comment:
Not a huge fan of this magic number. cf-layout already requires cf-grid, so we could calculate it based on cf-grid vars, right?

Reply:
Grid columns don't have a px value and we don't want to limit it to a % of the content area, just make sure it doesn't grow larger than 8 columns at max-width.